README.md

```diff
@@ -18,9 +18,5 @@ Todo
 - Design 
   - Figma
 
-### Android Studio Plugins
-- Mermaid\
-Visualizing graphs in markdown files
-
 ## Link
-[Figma](https://www.figma.com/design/glKpOCnBOjzx8mTTJUkyqH/Cyrcle?node-id=4-2&t=axZO6k6pPJuiWDin-1)
\ No newline at end of file
+[Figma](https://www.figma.com/design/glKpOCnBOjzx8mTTJUkyqH/Cyrcle?node-id=4-2&t=axZO6k6pPJuiWDin-1)
```

Comment:
You can remove the Figma from the ReadMe while you're at it.
The M1 requirement say it should be in the wiki, not in the readme anymore.
I think that changes make sense, so that all of our external ressources are in one place